.ci_support/linux_.yaml

```diff
@@ -0,0 +1,25 @@
+boost_cpp:
+- 1.70.0
```

Comment:
Curious, is there a way to generate this file from the cmake configs?  I worry that this file and the required minimums on the build would get out of sync...

Reply:
This file is generated by `conda smithy rerender` run in the root of the repo.  It applies the boost version that is currently being used across all conda-forge recipes in https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml as well as any local overrides we have in `recipe/conda_build_config.yaml`.   If it does get out of sync, it will be caught in CI.

I agree that we should figure out one place to store the required minimums.  I think that it will be easier to have them stored in a yaml configuration file that cmake can look at rather than having to get other things to understand where the minimums are kept in our cmake files.